trade-finance-manager-ui/server/controllers/utilisation-reports/utilisation-report-reconciliation-for-report/index.test.ts

```diff
@@ -128,32 +141,45 @@ describe('controllers/utilisation-reports/utilisation-report-reconciliation-for-
         },
       ];
 
-      const paymentDetailsViewModel: PaymentDetailsViewModel = [
-        {
-          payment: {
-            id: 1,
-            amount: { formattedCurrencyAndAmount: 'GBP 100.00', dataSortValue: 0 },
-            dateReceived: { formattedDateReceived: '1 Jan 2024', dataSortValue: 0 },
-            reference: undefined,
-          },
-          feeRecords: [{ id: 1, facilityId: '12345678', exporter: 'Test exporter' }],
-          feeRecordPaymentGroupStatus: FEE_RECORD_STATUS.MATCH,
-          reconciledBy: '-',
-          dateReconciled: { formattedDateReconciled: '-', dataSortValue: 0 },
-        },
-      ];
-
       const premiumPaymentsFilters = {
         facilityId: premiumPaymentsFacilityId,
       };
 
+      const paymentDetailsFilters = {
+        facilityId: paymentDetailsFacilityId,
+        paymentCurrency: paymentDetailsPaymentCurrency,
+        paymentReference: paymentDetailsPaymentReference,
+      };
+
+      const expectedPaymentDetailsViewModel: PaymentDetailsViewModel = {
+        rows: [
+          {
+            payment: {
+              id: 1,
+              amount: { formattedCurrencyAndAmount: 'GBP 100.00', dataSortValue: 0 },
+              dateReceived: { formattedDateReceived: '1 Jan 2024', dataSortValue: 0 },
+              reference: undefined,
```

Comment:
Not a blocker - would be nice to make `reference` optional so we don't have to pass `undefined` values around

Reply:
Pushed a change to the `PaymentDetailsPaymentViewModel` to make this optional thanks